Automattic/wp-calypso
# Stats: Date control update: Disable native calendar popup

Related to https://github.com/Automattic/red-team/issues/125

## Proposed Changes

* This PR disables the native calendar popup on the traffic page date picker, to allow us to use our different calendar instead. **Note: This does not display the correct date format that we want just yet, this is to come in a follow up PR**

## Why are these changes being made?


* it's part of a project to improve user experience for date selection

## Testing Instructions


* open up the Calypso live branch
* verify that the existing date control still works as expected without the feature flag
* apply `stats/date-picker-calendar` to the live branch
* verify that the date control no longer shows a calendar drop down
* verify that if you enter something which is not a date (eg a letter) it resets to today's date
* verify that typing in a new valid date in the correct format will load the correct date range chart data

![image](https://github.com/user-attachments/assets/d66851dd-9eb8-4089-be2d-2804254b842d)


## Pre-merge Checklist
